src/Serialization/Upgrades/1.3.0.jl

```diff
@@ -1,16 +1,105 @@
+function upgrade_NamedTuple(d::Dict)
```

Comment:
```suggestion
module SerializationUpgrade-1-3-0

function upgrade_NamedTuple(d::Dict)
```
It would be good to put all of these `upgrade_*` functions into their own shbmodule to not clutter the global namespace. And the next time someone needs to write an upgrade script, it could otherwise lead to name clashes.

Reply:
This was only temporary because I thought that revise couldn't handle updates inside the function that got pushed into the function list.
Also, it was unsafe because I was evalutating strings from files